docs/agents.md

```diff
@@ -8,9 +8,9 @@ but multiple agents can also interact to embody more complex workflows.
 The [`Agent`][pydantic_ai.Agent] class has full API documentation, but conceptually you can think of an agent as a container for:
 
 * A [system prompt](#system-prompts) — a set of instructions for the LLM written by the developer
-* One or more [retrievers](#retrievers) — functions that the LLM may call to get information while generating a response
+* One or more [tools](#tools) — functions that the LLM may call to get information while generating a response
```

Comment:
probably worth explicitly emphasizing that these will _not_ be used as the final response of the agent now that we are using terminology where that sometimes is the case